cypress/integration/Toast.spec.ts

```diff
@@ -29,9 +30,10 @@ describe('Toast', () => {
     });
   });
 
-  context(`given the toast with no close icon or action button`, () => {
+  context(`given the toast role of status`, () => {
```

Comment:
Updating this test context to refer to the Toast API (the `mode` prop) rather than the internals of the component (the `role` that's assigned to the rendered `aria-live` region).
```suggestion
  context(`given the toast set to the default mode of "status"`, () => {
```